packages/types/src/lib/interfaces.ts

```diff
@@ -1055,34 +1081,41 @@ export interface GetSignSessionKeySharesProp {
   body: SessionRequestBody;
 }
 export interface CommonGetSessionSigsProps {
+  /**
+   *
+   */
   pkpPublicKey?: string;
 
-  // When this session signature will expire.  The user will have to reauthenticate after this time using whatever auth method you set up.  This means you will have to call this signSessionKey function again to get a new session signature.  This is a RFC3339 timestamp.  The default is 24 hours from now.
+  /**
+   * When this session signature will expire. After this time is up you will need to reauthenticate, generating a new session signature. The default time until expiration is 24 hours. The formatting is an [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp.
+   */
   expiration?: any;
 
-  //   The chain to use for the session signature.  This is the chain that will be used to sign the session key.  If you're using EVM then this probably doesn't matter at all.
+  /**
+   * The chain to use for the session signature and sign the session key. This value is almost always `ethereum`. If you're using EVM, this parameter isn't very important.
+   */
   chain?: Chain;
 
   /**
    * An array of resource abilities that you want to request for this session. These will be signed with the session key.
-   *
-   * @example If you want to request the ability to decrypt an access control condition, then you would pass
-   * [{ resource: new LitAccessControlConditionResource('someResource), ability: LitAbility.AccessControlConditionDecryption }]
+   * If you want to request the ability to decrypt an access control condition, then you would pass something similar to the example:
+   * @example
+   * [{ resource: new LitAccessControlConditionResource('someResource`), ability: LitAbility.AccessControlConditionDecryption }]
    */
   resourceAbilityRequests: LitResourceAbilityRequest[];
 
   /**
    * The session capability object that you want to request for this session.
    * If you pass nothing, then this will default to a wildcard for each type of resource you're accessing.
    *
-   * @example If you passed nothing, and you're requesting to perform a decryption operation for an access
+   * If you passed nothing, and you're requesting to perform a decryption operation for an access
```

Comment:
I think the wording here is a bit confusing as we are explicitly outlining what will happen for all of our network options in a way which repeats language. Could we clean this up so we aren't repeating the same information?

Reply:
Changed first comment to:
> For example, an ability is added to grant a session permission to decrypt content associated with a particular Access Control Conditions (ACC) hash. When trying to decrypt, this ability is checked in the `resourceAbilityRequests` to verify if the session has the required decryption capability. 

The second comment is about `sessionCapabilityObject`, which going through the docs seemed very similar to `resourceAbilityRequests`. We don't have any example in the `developer-guides-code` or the Docs of using this, do we still encourage users to do so? Based on the description, a wildcard of each type of resource attempting to be accessed doesn't sound right. @joshLong145

Reply:
Yes you are correct in you assumption of being closely related to the `resourceAbilityRequests` we derive the `session capabilities` from the `resource request` you can see where we do this internally here: https://github.com/LIT-Protocol/js-sdk/blob/e66fb40a0d6ff269f25c6a06a7ad1cc00150326b/packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts#L281

This is not exposed since we derive from the given `ability requests` so hence the lack of examples.